### PR TITLE
updated Malware Analysis vocabulary definitions 

### LINKED ISSUE #294: Improve the definitions in Malware Result Vocabulary
10.13 Malware Result Vocabulary defines malicious, suspicious, benign, unknown but the definitions are not very descriptive. The STIX WG on Oct. 4, 2022 requested that we consider improve these definitions in the next version.

### PR DESCRIPTION
**Pull request by ejratl**
This PR incorporates the Malware analysis vocabulary definitions suggested by Sean Carroll. Fixes #294

### CODE REVIEW

**Comment by adulau** [comment]
Looks great to me. Thank you. Maybe someone with write access could merge it...

**Comment by ejratl** [comment]
Thanks for your review @adulau ! I believe that this will be merged during the next STIX WG meeting.

File: spec/drafts/v2.1.1/stix-v2.1.1.adoc [hunk 1]
```diff
@@ -10993,10 +10993,10 @@ This is a non-exhaustive, open vocabulary that captures common types of scanner
...
-|[stixliteral]#benign# |The tool reported the malware binary as benign.
-|[stixliteral]#unknown# |The tool was unable to determine whether the malware binary is malicious.
+|[stixliteral]#malicious# |The tool or human analysis determined that the sample is designed to operate, execute or take place in a manner that is not expected by legitimate users, or performs one or more actions generally deemed harmful to a system, or the legitimate users of a system. These can take the form of executables, source code, scripts or any other software or commands.
+|[stixliteral]#suspicious# |The tool or human analysis determined that the sample does not operate as expected or is is usually present in conjunction with a malicious file. But does not itself demonstrate malicious behaviors. Examples may include files not expected to be present or that support applications that have not been installed, files with an incorrect attribute (location, version, size), or which are accessed or loaded with unusual frequency or at unusual times. Other examples include files dropped or created when malware runs.
```
Comment (dzbeck) [inline spec/drafts/v2.1.1/stix-v2.1.1.adoc:10997]: typo - two "is" in first sentence. Also, one sentence needs a subject - suggest, "But **the file** does not itself..."

Comment (dzbeck) [reply spec/drafts/v2.1.1/stix-v2.1.1.adoc:10997]: Looks great to me too - a nice update!

**Comment by dzbeck** [comment]
Looks great to me too - a nice update!

**Pull request merged**